spring-data-cassandra/src/test/java/org/springframework/data/cassandra/repository/support/BasicMapIdUnitTests.java

```diff
@@ -16,22 +16,36 @@
 
 package org.springframework.data.cassandra.repository.support;
 
-import static org.assertj.core.api.Assertions.*;
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertSame;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
 
 import java.io.Serializable;
+import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Set;
 
 import org.junit.Test;
+import org.springframework.data.cassandra.repository.MapId;
 
 /**
  * Unit tests for {@link BasicMapId}.
  *
  * @author Matthew T. Adams
+ * @author Michael Hausegger
  */
 public class BasicMapIdUnitTests {
+
```

Comment:
Please use the [Spring Data formatting settings (imports and code style)](https://github.com/spring-projects/spring-data-cassandra/tree/master/etc).